### ISSUE TITLE
Accessiblity: Missing 'alt text' for images

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/Ottawa-ckan#24]
WCAG 2.0 Success Criteria 1.1.1 - Non text Content  Level (A) Compliance violation - All images must have equivalent alternative text.
1. Go to: data.ottawa.ca
2. Observe for alt text on the images within the 'Outdoor Pools' and 'Parks and Volleyball Courts' images.

Resolution: Add alternative text to images.
![missing alt text images](https://cloud.githubusercontent.com/assets/5271253/8010763/0460fc6c-0b81-11e5-9454-4526d4eb5f4d.JPG)
![Uploading Volleyball courts alt text.JPG…]()

**Comment by StephenOTT** [comment]
The best practice for dealing with maps and accessibility is to provide a alternative to the map such as a Table rather than trying to make the map accessible.  Making a map "accessible" is not a easy task and is a endless process of  adjustments and thinking about how the map is actually used (aka what is the purpose of the map to the user: What is the actual task they would be accomplishing by looking at the map)
Featherstone does a good talk about this issue:
http://store.simplyaccessible.com/virtual-seminar/accessible-maps/

**Comment by deniszgonjanin** [comment]
@StephenOTT thanks, that's very helpful. Most of these maps are used to show raw data, instead of a specific function such as providing directions to a particular landmark, etc... Showing that data in a tabular format instead of a map would not provide much better accessibility.

I think for now we'll set the Alt Text of the map to the Description of the dataset to which it belongs to. That might be semantically the most meaningful thing we can do, as it provides users with screen readers with the context of what the map is actually about. 

P.S. This issue has been moved to https://github.com/CityofOttawa/ckanext-ottawa/issues/4 by the way, if you want to follow it
